tests/sources2csr/test_sources2csr.py

```diff
@@ -58,3 +65,27 @@ def test_missing_identifier(tmp_path):
     with pytest.raises(DataException) as excinfo:
         reader.read_subject_data()
     assert 'Empty identifier' in str(excinfo.value)
+
+
+def test_wrong_file_format(tmp_path):
+    target_path = tmp_path.as_posix()
+    reader = SourcesReader(
+        input_dir='./test_data/input_data/CLINICAL',
+        output_dir=target_path,
+        config_dir='./test_data/input_data/config/invalid_sources_config/wrong_file_format')
+    with pytest.raises(DataException) as excinfo:
+        reader.read_subject_data()
+    # The columns in the biosource.csv file are not correctly parsed, resulting
+    # in a missing identifier
+    assert 'Empty identifier' in str(excinfo.value)
```

Comment:
I wonder if there is a way to return some better tips as an error message, when there is an issue with a file_format configuration (typo in a file name etc.). It's easy to forget about checking this.

Reply:
Changed it to missing column with a hint at the sources config.